Tests/ForageSDKTests/MaskedUITextFieldTests.swift

```diff
@@ -20,15 +20,15 @@ final class MaskedUITextFieldTests: XCTestCase {
     
     override func tearDown() {
         maskedTextField = nil
+        ForageSDK.shared.environment = .sandbox
```

Comment:
Why is this set on teardown?

Reply:
Because the `environment` is set to `.prod` in one of the tests. The `ForageSDK.setup` function doesn't update the environment value.

I've updated the tests here and added a comment: f92739a8ba22312b9f54e99e704d99738142a05a

```swift
    public class func setup(_ config: Config) {
        ForageSDK.config = config
       // we never update self.environment = ... here
    }
```

The fix is addressed in #75, which ensures the environment can be updated as desired (using the latest session token).